models/House.js

```diff
@@ -1,29 +1,73 @@
-const mongoose = require("mongoose");
+const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
-const User = require("./User");
-const Bill = require("./Bill");
+const User = require('./User');
+const Bill = require('./Bill');
 
 const houseSchema = new Schema({
-  owner: { type: Schema.ObjectId, ref: "User" },
+  owner: {
+    type: Schema.ObjectId,
+    ref: "User"
+  },
   account: {
-    bsb: Number,
-    accNumber: Number
+    bsb: {
+      type: Number,
+      required: [true, 'BSB number required'],
+      validator: function (v) {
+        return /\d{2} \d{4}/.test(v);
```

Comment:
Pemisahnya enak spasi ato dash? @demounit

Reply:
Gw awalnya mw taruh dash for the separator, tp di website netbank, pemisahnya pake spasi. (tho I prefer dash too)

Reply:
Alright, go with 'space' 👍